packages/kbn-securitysolution-grouping/src/components/accordion_panel/index.tsx

```diff
@@ -41,9 +40,11 @@ const DefaultGroupPanelRenderer = ({
 }) => (
   <div>
     <EuiFlexGroup gutterSize="s" alignItems="center" responsive={false}>
-      <EuiFlexItem grow={false}>
+      <EuiFlexItem grow={false} className="eui-textTruncate">
```

Comment:
our truncate was never working properly and this issue brought that to my attention, thus fixed here